Libraries/BatchedBridge/MessageQueue.js

```diff
@@ -52,8 +52,8 @@ const guard = (fn) => {
 class MessageQueue {
   _callableModules: {[key: string]: Object};
   _queue: [Array<number>, Array<number>, Array<any>, number];
-  _callbacks: [];
-  _callbackID: number;
+  _successCallbacks: Array<?Function>;
```

Comment:
Optional because they're set to null once called, Function rather than anything more specific because we don't know how many args they're called with, afaik. Open to suggestions?